public/app/core/components/PageNew/SectionNavItem.tsx

```diff
@@ -36,9 +37,17 @@ export function SectionNavItem({ item, isSectionRoot = false }: Props) {
     icon = <Icon data-testid="section-icon" name={item.icon} />;
   }
 
+  const onItemClicked = () => {
+    reportInteraction('grafana_navigation_item_clicked', {
+      path: item.url ?? item.id,
+      sectionNav: true,
```

Comment:
@yaelleC i made this `sectionNav` instead of `secondaryNav` as that's what it's called in our code internally. but can absolutely change this to `secondaryNav` if that's incorrect

Reply:
currently in the `grafana_navigation_item_clicked` event we have a `state: expanded/collapsed` I suppose this is not applicable for top nav?
Happy to leave it out if not needed, just checking 🙂